pkg/apis/sriovnetwork/v1/sriovibnetwork_types.go

```diff
@@ -0,0 +1,61 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// SriovIBNetworkSpec defines the desired state of SriovIBNetwork
+type SriovIBNetworkSpec struct {
```

Comment:
There are other config fields for the ib-sriov, can you add them, i.e. `pkey`, `rdmaIsolation` and `ibKubernetesEnabled`